docs/API Reference/API-Reference.md

```diff
@@ -32,20 +32,19 @@ In case you run an on-premise version then the URL will correspond with its URL
 
 ---
 
-## When to use REST and GraphQL
-Since Easy Software offers both, REST API and GraphQL, it is important to mention why that is.
-REST API is used for back end to back end communication, so for the cases when some data needs to get from you system to Easy Software and vice versa.
+## When to use "REST-like API" and GraphQL
+Since Easy Software offers both, HTTP API (json, xml) and GraphQL, it is important to mention why that is.
+Regular API is used for back end to back end communication, so for the cases when some data needs to get from you system to Easy Software and vice versa.
 Contrary to that, GraphQL is used for communication between the front end and back end of Easy Software and thus can be used if you wish to implement your own plugin for it.
-
 ---
 
 ## Rate limits
 There is a limit of requests you can send to the cloud version of Easy Software. This limit is set to `10` requests per minute which when reached triggers 60 second cooldown during which you will receive `HTTP status code 429 - Rate limit exceeded`.
 
 ---
 
-## REST request and response formats
-Easy Software REST API supports sending and receiving messages in JSON and XML format. The format you want to provide and consume has to specified for every call as a path parameter just like this:
+## request and response formats
+Easy Software API supports sending and receiving messages in JSON and XML format. The format you want to provide and consume has to specified for every call as a path parameter just like this:
```

Comment:
Did you agree on using the company name for some reason? 
Is sounds odd. A product name would be more suitable.
I'm not sure what form to use tho. 'EP/ER' vs 'Easy Project / Easy Redmine' vs just 'Easy Project'. What do you think?
```suggestion
EP/ER API supports sending and receiving messages in JSON and XML format. The format you want to provide and consume has to specified for every call as a path parameter just like this:
```